test/integration/targets/group/tasks/tests.yml

```diff
@@ -211,7 +211,18 @@
     - user_test_local_mode
 
 - name: Ensure lgroupadd is present - Alpine
-  command: apk add -U libuser --repository=http://dl-cdn.alpinelinux.org/alpine/edge/community
+  # NOTE: `quay.io/ansible/alpine3-test-container:2.0.2` does not have
```

Comment:
@mattclay do you think this 2.11-only change needs an updated comment wording too?

Reply:
I would. The issue affects anyone using an older Alpine system, not just those using our test container.

Reply:
@mattclay fixed